lib/src/components/Loader/theme.ts

```diff
@@ -1,11 +1,10 @@
 import { ThemeValues } from '@/theme'
+import { LiteralUnion } from '@/utils'
 
-export type ThemeLoaders = {
-  lg: string
-  md: string
-  sm: string
-  xs: string
-}
+type LoarderSize = 'xs' | 'sm' | 'md' | 'lg'
+export type Size = LiteralUnion<LoarderSize, string | number>
+
+export type ThemeLoaders = Record<LoarderSize, string>
```

Comment:
```suggestion
type LoaderSize = 'xs' | 'sm' | 'md' | 'lg'
export type Size = LiteralUnion<LoaderSize, string | number>

export type ThemeLoaders = Record<LoaderSize, string>
```

Sorry to be so pedantic 🤓

Reply:
Good catch! I should go back to school 😭